packages/timeline-state-resolver-types/src/vizMSE.ts

```diff
@@ -177,16 +179,16 @@ export interface TimelineObjVIZMSEInitializeShows extends TSRTimelineObjBase {
 		type: TimelineContentTypeVizMSE.INITIALIZE_SHOWS
 
 		/** IDs of the Shows to initialize */
```

Comment:
Would be an idea to update `IDs` to `Names` here as well, just as you did further up :)
Same for line 191.